### ISSUE TITLE
Train own datasets（BelgiumTS Dataset    ）

### ISSUE THREAD

**Issue by sumenpuyuan** [balancap/SSD-Tensorflow#244]
Hi！Everyone！
I want to train my dataset，the url of data ishttp://btsd.ethz.ch/shareddata/，it has 62 classes。
I have recently converted the data to the voc07 format。so on terminal
`DATASET_DIR=./trafic_records
TRAIN_DIR=./logmine/
CHECKPOINT_PATH=./checkpoints/vgg_16.ckpt
python train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=train \
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --checkpoint_model_scope=vgg_16 \
    --checkpoint_exclude_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --trainable_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --save_summaries_secs=60 \
    --save_interval_secs=600 \
    --weight_decay=0.0005 \
    --optimizer=adam \
    --learning_rate=0.001 \
    --learning_rate_decay_factor=0.94 \
    --batch_size=32`
the loss is about 20.but mAP is close to zero，and can't detection on the jupyter notebook
I would like to ask if anyone has successfully trained his own data set. What are the specific steps?
My qq is 1508462296, wechat is sumenpuyuan
Many thanks!

**Comment by sumenpuyuan** [comment]
I succeeded in half, I reduced the 62 data sets of the data set to 10, then changed the pascalvoc_common.py file, and finally succeeded in identifying, but there are many problems with 62. I suspect that as long as there are less than 21 classes, there is basically no need to change the source code.
So if there are many types of datasets, such as 62 categories, which areas should I modify specifically, does anyone know?

**Comment by vokhidovhusan** [comment]
@sumenpuyuan  I am facing the same problem. I succeeded with 20 classes but can't detect if classes are more than 20. Any solutions?

**Comment by nizhanzhu** [comment]
Hello,I want to train my dataset,If I have a image which have no object,do i need to give a object name (eg.background ) and  box in XML file or just a objectname .What about  pascalvoc2007?if it add a backgroud class,do it represent there is a image have no other class,just a object name called bacground.Maybe it have not image having no object ?
